driver-pulsar/src/main/java/io/nosqlbench/driver/pulsar/ops/ReadyPulsarOp.java

```diff
@@ -461,7 +461,7 @@ else if (StringUtils.equalsIgnoreCase(stmtOpType, PulsarActivityUtil.OP_TYPES.E2
             try {
                 nsMsgDedup = pulsarAdmin.namespaces().getDeduplicationStatus(namespace);
```

Comment:
We shouldn't allow NPEs to be swallowed this way. Instead, get the higher level object first and check for null.
It's just too risky to swallow exceptions like this. It's not entirely clear that an NPE is the only way this could error, and also that that is always ok.